### PR TITLE
Documentation change for postgresql_owner.py 

### PR DESCRIPTION
**Pull request by jchancojr**
##### SUMMARY
Update documentation to reflect what actually happens when `REASSIGNED OWNED BY` is invoked. 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
`postgresql_owner`

### CODE REVIEW

File: changelogs/fragments/issue_240_doc_change.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- postgresql_owner - Change documentation to specify that ``REASSIGNED OWNED BY`` will impact the entire cluster/instance and not just a single database.
```
Comment (Andersson007) [inline changelogs/fragments/issue_240_doc_change.yml:2]:
```suggestion
- postgresql_owner - Change documentation to specify that ``REASSIGNED OWNED BY`` will impact the entire cluster/instance and not just a single database (https://github.com/ansible-collections/community.postgresql/issues/240). 
```

**Comment by Andersson007** [comment]
fixes https://github.com/ansible-collections/community.postgresql/issues/240

**Comment by Andersson007** [comment]
@jchancojr thanks for the PR!
I usually add "Fixes <link-to-issue>" to an issue description so that it'll get closed automatically after a corresponding PR gets merged (it could be in a comment also IIRC)

**Comment by Andersson007** [comment]
Ah, sorry, we don't need changelog fragments for doc PRs, so could you please remove it?

**Review by hunleyd — APPROVED** [review]

**Pull request merged**